rest-api/tools/ehr_upload.py

```diff
@@ -0,0 +1,78 @@
+    get_participants_under_sites(
+      session, args.organization, args.source_bucket, args.destination_bucket)
+
+
+def get_participants_under_sites(session, organization, source_bucket, destination_bucket):
+  sql = """
+  select p.participant_id, google_group from participant p left join site s on p.site_id = s.site_id left join participant_summary summary on p.participant_id = summary.participant_id
+  where p.organization_id in (select organization_id from organization where external_id = '{}')
+  and summary.consent_for_electronic_health_records =1 and summary.consent_for_study_enrollment = 1;
+   """.format(organization)
+  cursor = session.execute(text(sql))
+  try:
+    results = cursor.fetchall()
+    results = [(int(i), str(k)) for i, k in results]
+    if results:
+      for participant, google_group in results:
+        gsutil = "gsutil -m cp gs://" + source_bucket + "/Participant/P" + str(
+          participant) + "/* " + "gs://" + destination_bucket + "/Participant/" + \
+          google_group + "/P" + str(participant)
+
+        system_call = subprocess.call(gsutil, shell=True)
+        if system_call == 0:
+          print "Successfully moved folder " + google_group + '/' + str(participant)
+        elif system_call == 1:
+          print "There was an error moving folder " + google_group + '/' + str(participant)
+  finally:
+    cursor.close()
+    get_participants_without_site_pairing(session, organization, source_bucket, destination_bucket)
+
+
+def get_participants_without_site_pairing(session, organization, source_bucket, destination_bucket):
+  sql = """
+    select participant_id from participant p
+    where p.site_id is NULL and p.organization_id in (
+    select organization_id from organization where external_id = '{}');
+   """.format(organization)
+  cursor = session.execute(text(sql))
+  try:
+    results = cursor.fetchall()
+    if results:
+      results = [int(i) for i, in results]
+      for participant in results:
+        gsutil = "gsutil -m cp gs://" + source_bucket + "/Participant/P" + str(
+          participant) + "/* " + "gs://" + destination_bucket + "/Participant/" + \
+                 "no_site_pairing" + "/P" + str(participant)
+
+        system_call = subprocess.call(gsutil, shell=True)
+        if system_call == 0:
+          print "Successfully moved folder " + "no_site_pairing" + '/' + str(participant)
+        elif system_call == 1:
+          print "There was an error moving folder " + "no_site_pairing" + '/' + str(participant)
+  finally:
+    cursor.close()
+
+
+if __name__ == '__main__':
+  configure_logging()
+  parser = get_parser()
+  parser.add_argument('--organization', help='The organization to find participants and sites for',
+                      required=True)
+  parser.add_argument('--source_bucket', help='The bucket to read from in one env.'
+                      ' i.e. ptc-uploads-pmi-drc-api-prod', required=True)
```

Comment:
Is it ok for these to have the same value? Is that typical?

Reply:
I changed it.